applications/metaquity_nft_provenance_tracker.md

```diff
@@ -0,0 +1,148 @@
+
+Technology stack: 
+- MongoDB 
+- Typescript 
+- Elasticsearch
+- JavaScript
+- [Rust-web3](https://github.com/tomusdrw/rust-web3)
+- Nginx, AWS
+
+### Ecosystem Fit
+
+There are quite a few projects in the Polkadot/Kusama ecosystem that are working with NFTs. And going forward with XCM v4 supporting cross-chain transfer of NFT's, we believe tracking and tracing of the entire lifecycle of an NFT will not only be a need but a must have.
+
+The NFT-Provenance-Tracker SDK built by Metaquity will be a great way for these projects to implement provenance for their projects and will save a lot of time and effort to build this from scratch.
+
+Also in line with our project, we intend to extend this SDK to include use cases of tracking fractional ownership of the individual NFTs. This use case alone will open a lot of doors of bringing real world assets into the Polkadot ecosystem.
+
+Later on we can extend EVM compatibility to this project to make it more versatile.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Abhishek Chatterjee
+- Abhishek Sira Chandrashekhar
+- Arko Ganguli
+
+### Contact
+
+- **Contact Name:** Abhishek Chatterjee
+- **Contact Email:** abhishek@metaquity.xyz
+- **Website:** [Metaquity](www.metaquity.xyz/)
+
+### Legal Structure
+
+- **Registered Address:** 128 CITY ROAD, UNITED KINGDOM, LONDON, EC1V 2NX
+- **Registered Legal Entity:** Metaquity Limited.
+
+### Team's experience
+
+- **Abhishek Chatterjee**: Co-founder of Metaquity, 15+ years of product and technical experience. 4+ years of blockchain experience. 
+- **Abhishek Sira Chandrasekhar**: Software developer with 4+ years of experience in development 2+ years in Blockchain working on DID, SSI, VC and ZKP. Prior to Metaquity used to work at Litentry as a mobile developer.
+
+- Metaquity Network is already part of the Substrate Builders Program.
+
+
+### Team Code Repos
+
+- https://github.com/metaquiry-network/nft-provenance-tracker
+
+Team members GitHub:
+
+- https://github.com/abhichat85
+- https://github.com/ekaanth
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/abhishekchatterjee85/
+- https://www.linkedin.com/in/abhishek-sira-chandrashekar-821244b7/
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 2,5 months
+- **Full-Time Equivalent (FTE):**  2 FTE (full-time employees)
+- **Total Costs:** 20,000 USD
+
+### Milestone 1 
+
+- **Estimated duration:** 1 month
+- **FTE:**  1,5
+- **Costs:** 10,000 USD
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License |   Apache 2.0 |
+| **0b.** | Documentation | Architectural documentation of NFT-scanner |
+| 1. | Data model definition | Designing and defining the architectural overview of the data model for this application  |
+| 2. | NFT scanner module | This module will scan the blockchain for any NFT-related data from the genesis block and invoke the indexer module |
```

Comment:
In which layer of the stack will this module be located? What tech is going to be used here?

Reply:
[Quote](https://github.com/w3f/Grants-Program/pull/1832/files#diff-d261c1c087023818a19116461c698e206ce055aaa2b6591bee8e1f60b682bdd2R109) from your proposal:

> This module will be in middle layer to scan the blockchain for the blocks containing NFT related data from the genesis block and invoke indexer module (We will be indexing only blocks that has NFT related data.)

By "middle layer" you mean the "middleware layer" as depicted in your [architecture diagram](https://github.com/w3f/Grants-Program/pull/1832/files#diff-d261c1c087023818a19116461c698e206ce055aaa2b6591bee8e1f60b682bdd2R21)? Could you distinguish which technologies are used in which layer and how (protocols or other means of communication) they interact with each other?

Reply:
The middleware layer is written mostly in typescript; they communicate or interact with the data using JSON.